hbase_storage/src/main/java/com/splicemachine/si/impl/CoprocessorTxnStore.java

```diff
@@ -395,12 +397,21 @@ private TxnView decode(long queryId, TxnMessage.Txn message) throws IOException{
             // we didn't find it
             if (ignoreMissingTransactions) {
                 // return committed mock transaction
-                return new InheritingTxnView(Txn.ROOT_TRANSACTION,queryId,queryId,
+                return new InheritingTxnView(Txn.ROOT_TRANSACTION, queryId, queryId,
```

Comment:
Another set of changes that I don't recognize from the 2.8 and Master versions of this PR. Are these supposed to be here in 3.0?